packages/components/src/sectorDetailsWrapper/SectorDetailsWrapper.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import * as React from 'react';
+import { Touchable, Icon, StyleSheet } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Animated } from 'react-native';
+
+type Props = {|
+  +children: React.Node,
+|};
+
+type State = {|
+  +expandSegments: boolean,
+  +isDisabled: boolean,
+|};
+
+export default class SectorDetailsWrapper extends React.Component<
+  Props,
+  State,
+> {
+  constructor() {
+    super();
+
+    this.state = {
+      expandSegments: false,
```

Comment:
I'm not sure about name of this key. What about `areSegmentExpanded` or something like that?

Reply:
Right. I think we are using `isSomething` across project so I'll change it to `isExpanded`.

Reply:
@jvaclavik  Updated